sql/moz-fx-data-shared-prod/telemetry/clients_last_seen_v1/view.sql

```diff
@@ -61,6 +66,46 @@ SELECT
       ) <= 1
     )
   ) AS is_allweek_regular_v1,
+  BIT_COUNT(days_visited_1_uri_bits) >= 21 AS is_core_active_v1,
+  CASE
+  WHEN
+    BIT_COUNT(days_visited_1_uri_bits)
+    BETWEEN 1
+    AND 6
+  THEN
+    'infrequent_user'
+  WHEN
+    BIT_COUNT(days_visited_1_uri_bits)
+    BETWEEN 7
+    AND 13
+  THEN
+    'casual_user'
+  WHEN
+    BIT_COUNT(days_visited_1_uri_bits)
+    BETWEEN 14
+    AND 20
+  THEN
+    'regular_user'
+  WHEN
+    BIT_COUNT(days_visited_1_uri_bits) >= 21
+  THEN
+    'core_user'
+  ELSE
+    'other'
+  END
+  AS activity_segments_v1,
+  days_since_created_profile = 6
```

Comment:
@irrationalagent Now that first_seen_date is available here, do you want to recast these to use that rather than `days_since_created_profile`? Or should we stick with this for now and consider updating later?

Reply:
How much pain would it be to change later? IMO we should probably change, but I know people rely on this field and I'm not sure if there are use cases that would break by us changing without consulting first.

What's the pain in having both `days_since_created_profile` and `days_since_first_seen`? I believe having both is probably preferable even if it causes some confusion (though I think pretty much all DS already knows the difference)

Reply:
I wasn't clear in the scope of what I was proposing here. I definitely would want to keep both days_since_created_profile and days_since_first_seen. `days_since_created_profile` would continue to be based on profile_creation_date.

My intent was to suggest that we might want to define the new `new_profile_7_day_activated_v1`, etc. fields in terms of `days_since_first_seen` rather than `days_since_created_profile`.

Reply:
Oops, I see. We tend to enroll users in experiments using PCD, so I think we would probably want to keep it that way for now to avoid any mismatch there.

Reply:
@jklukas After canvassing DS a bit, I retract my previous comment. Let's go ahead and use `first_seen_date` when calculating `new_profile_7_day_activated_v1`. Sorry for the noise

Reply:
Done.